src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -207,8 +209,17 @@ public void testFindAll_noRowsFound() {
 
   @Test
   public void testFindAll_limitOnly() {
-    final List<DatasetRow> newDatasetRows =
-        newDatasetRowsWith(namespaceRow.getUuid(), datasourceRow.getUuid(), 4);
+    List<DatasetRow> newDatasetRows = new ArrayList<>();
```

Comment:
Can we add a test case where we insert datasets in a non-sorted order and verify that they come back sorted?

Reply:
Sure

Reply:
I took care of this case in both the test cases which inserts the randomly generated list first and then sorts it and compares.